server/src/Config.hs

```diff
@@ -54,6 +59,7 @@ type EnvironmentConfig = [(String, String)]
 
 configParser :: EnvironmentConfig -> Parser Config
 configParser environment = Config
+  -- TODO: We should use a different default ("icepeak.db") when SQLite is chosen as the storage backend
```

Comment:
We might handle this elsewhere then, and not have a default in the parser? The option would be optional then.